azkaban-common/src/main/java/azkaban/imagemgmt/models/ImageOwnership.java

```diff
@@ -32,60 +35,74 @@
   private Role role;
 
   public String getName() {
-    return name;
+    return this.name;
   }
 
-  public void setName(String name) {
+  public void setName(final String name) {
     this.name = name;
   }
 
   public String getOwner() {
-    return owner;
+    return this.owner;
   }
 
-  public void setOwner(String owner) {
+  public void setOwner(final String owner) {
     this.owner = owner;
   }
 
   public Role getRole() {
-    return role;
+    return this.role;
   }
 
-  public void setRole(Role role) {
+  public void setRole(final Role role) {
     this.role = role;
   }
 
   /**
-   * Enum representing owner Role
-   * ADMIN role is having all the permissions such as invoking image management APIs, Adding new
-   * job type owner etc.
-   * MEMBER - Must have permissions to invoke image management APIs, but can't add/delete job
-   * type owners etc. However the MEMBER role needs to be clearly defined.
+   * Enum representing owner Role ADMIN role is having all the permissions such as invoking image
+   * management APIs, Adding new job type owner etc. MEMBER - Must have permissions to invoke image
+   * management APIs, but can't add/delete job type owners etc. However the MEMBER role needs to be
+   * clearly defined.
    */
   public enum Role {
     ADMIN("admin"),
     MEMBER("member");
```

Comment:
Was there any need of Read only (GUEST) role over here? I still think that you should use enum name itself instead of introducing name as instance variable.

Reply:
Added guest role.